src/extra-fields/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import metadata from './block.json';
+import './style.scss';
```

Comment:
```suggestion
import metadata from './block.json';
import './style.scss';

const save = () => null;
```
Let's keep it local for as long as we don't need it.